<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>910</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>910</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WinCom</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/resource/wincom.ico</normaloff>:/icon/resource/wincom.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>串口设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="11" column="0">
          <widget class="QComboBox" name="flowControlComboBox">
           <item>
            <property name="text">
             <string>OFF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HW_CTRL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SW_CTRL</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_port">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>端口号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="portNameComboBox">
           <item>
            <property name="text">
             <string>COM0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>COM15</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_baudrate">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="baudRateComboBox">
           <property name="currentText">
            <string>1200</string>
           </property>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_databits">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="dataBitsComboBox">
           <property name="currentText">
            <string>5</string>
           </property>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_parity">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>校验位</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QComboBox" name="parityComboBox">
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>偶</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>奇</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_stopbits">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>停止位</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QComboBox" name="stopBitsComboBox">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_flow">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>流控制</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="openMyComBtn">
          <property name="text">
           <string>打开串口</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icon/resource/open.png</normaloff>:/icon/resource/open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="write2fileButton">
          <property name="text">
           <string>写入..</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icon/resource/write2file.png</normaloff>:/icon/resource/write2file.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeMyComBtn">
          <property name="text">
           <string>关闭串口</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icon/resource/stop.png</normaloff>:/icon/resource/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearShowBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>清空显示</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icon/resource/cleanport.png</normaloff>:/icon/resource/cleanport.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitBtn">
          <property name="text">
           <string>退出程序</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icon/resource/exit.png</normaloff>:/icon/resource/exit.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_movie">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>接收数据</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="ccradioButton">
             <property name="text">
              <string>字符形式</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="chradioButton">
             <property name="text">
              <string>HEX形式</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>72</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>已读取字节数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="recvbyteslcdNumber">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(85, 255, 127);</string>
             </property>
             <property name="smallDecimalPoint">
              <bool>false</bool>
             </property>
             <property name="digitCount">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearBytesBtn">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>计数清零</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icon/resource/clearbytes.png</normaloff>:/icon/resource/clearbytes.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTextBrowser" name="textBrowser">
           <property name="minimumSize">
            <size>
             <width>650</width>
             <height>290</height>
            </size>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>发送数据</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButton_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>十六进制发送</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="sendMsglineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="sendMsgBtn">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>发送数据</string>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/icon/resource/send.png</normaloff>:/icon/resource/send.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionAddPort"/>
    <addaction name="separator"/>
    <addaction name="actionOpenPort"/>
    <addaction name="actionClosePort"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionWriteToFile"/>
    <addaction name="separator"/>
    <addaction name="actionClearBytes"/>
    <addaction name="actionCleanPort"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查找</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionFindPre"/>
    <addaction name="actionFindNext"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddPort">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/add.png</normaloff>:/icon/resource/add.png</iconset>
   </property>
   <property name="text">
    <string>添加串口</string>
   </property>
   <property name="toolTip">
    <string>添加串口设备</string>
   </property>
  </action>
  <action name="actionOpenPort">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/open.png</normaloff>:/icon/resource/open.png</iconset>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </action>
  <action name="actionClosePort">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/stop.png</normaloff>:/icon/resource/stop.png</iconset>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </action>
  <action name="actionCleanPort">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/cleanport.png</normaloff>:/icon/resource/cleanport.png</iconset>
   </property>
   <property name="text">
    <string>清空串口</string>
   </property>
   <property name="toolTip">
    <string>清空串口中I/O数据</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/save.png</normaloff>:/icon/resource/save.png</iconset>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
   <property name="toolTip">
    <string>保存收发框的数据</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/exit.png</normaloff>:/icon/resource/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/mesage.png</normaloff>:/icon/resource/mesage.png</iconset>
   </property>
   <property name="text">
    <string>关于WinCom</string>
   </property>
  </action>
  <action name="actionWriteToFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/write2file.png</normaloff>:/icon/resource/write2file.png</iconset>
   </property>
   <property name="text">
    <string>写入..</string>
   </property>
   <property name="toolTip">
    <string>将读取的数据写入到文件</string>
   </property>
  </action>
  <action name="actionClearBytes">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/clearbytes.png</normaloff>:/icon/resource/clearbytes.png</iconset>
   </property>
   <property name="text">
    <string>计数清零</string>
   </property>
   <property name="toolTip">
    <string>清零已收到的字节数</string>
   </property>
  </action>
  <action name="actionFindPre">
   <property name="text">
    <string>查找上一个</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>查找下一个</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/resource/find.png</normaloff>:/icon/resource/find.png</iconset>
   </property>
   <property name="text">
    <string>查找...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
